### ISSUE TITLE
Request - Automatic "use_portal" by default

### ISSUE THREAD

**Issue by Khormin** [Raphfrk/ServerPort#23]
Any way to toggle it from a whitelist on using the server portal, to a blacklist?

**Comment by Raphfrk** [comment]
That isn't currently supported, I need to re-do server port permissions in general though.

**Comment by Khormin** [comment]
No problems.  The "newplayers" toggle also does nothing.  All players are being booted until manually added.

**Comment by Khormin** [comment]
I'll take a look at the java settings on server, but I suspect it's just ServerPort that's doing it.

**Comment by Raphfrk** [comment]
What does it say when they are kicked?

**Comment by Khormin** [comment]
I'll take a look in a bit, making dinner atm.  More or less "ServerPort
Newplayers is set to TRUE, you will be kicked from server if you move"/same
with "set to FALSE"

On 28/08/2011 7:30 PM, "Raphfrk" <
reply@reply.github.com>
wrote:

**Comment by Khormin** [comment]
Here we go.  Had a nightmare getting it to replicate, deleting myself from the Limbo directory didn't do anything.

"[Serverport] New players are not allowed to connect to this server directly.  If you move you will be teleported"

It does this until the plugin folder Limbo directory gets a new file for that particular player, regardless of newplayers setting toggle.  No setting, TRUE, and FALSE all do it.

**Comment by Raphfrk** [comment]
Looking at the code, the parameter needs to be set to "allow" to allow new players.  Otherwise, it should be set to the server that you want to redirect the players to.

**Comment by Khormin** [comment]
Hmm.  So it's not a 'true/false' toggle?  It suggests true/false in the
/serverport help

On 29/08/2011 7:22 PM, "Raphfrk" <
reply@reply.github.com>
wrote:

**Comment by Raphfrk** [comment]
Yeah, I should update the help info.  It was true/false, but somebody asked for a redirect option.

**Comment by Raphfrk** [comment]
Hmm, the help comment is:

```
Unless this is set to allow, new players will be redirected to this server
```

I guess I could change it to

```
Set this to the name of the server to redirect new players to, or "allow" if new players are allowed
```

**Comment by Khormin** [comment]
That would work.  Or

Set to "allow", or give the name of the server that you would rather new players connect to.

Though I'm still seeing "Unless set to true, new players will be unable to connect".  Might have to double-check my copy is up to date.
